### ISSUE TITLE
[graphics][desktop] Why KDE Plasma is better than GNOME

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/rootpages#1066]
```
Nobara Official has now be rebased on KDE instead of GNOME for several reasons:
———–
(1) VRR (Variable Refresh Rate/Freesync) functionality. Gnome currently still requires patched-in VRR functionality. The pending MR has been waiting 3 years, and even doing so still breaks the system tray.

See:
https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1154
https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6955

(2) DRM Leasing functionality. This is a requirement for VR (Virtual Reality) in wayland. This is another pain point that has had constant back and forth on the gnome side, but is implemented already in KDE.

See:
https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/3205
https://gitlab.gnome.org/GNOME/mutter/-/issues/1743

(3) Fractional scaling functionality. This is what allows the desktop to be rescaled in fractions rather than whole percentages. This is still marked as experimental in GNOME even though it was merged in 2019, 5 years ago, while being implemented already in KDE.

(4) Drag and drop from Archive manager functionality. As stupid as this is, GNOME file-roller STILL cannot drag+drop files into nautilus folders, while KDE’s Ark can. Again, another bug opened 5 years ago.

See:
https://gitlab.gnome.org/GNOME/file-roller/-/issues/4

(5) Better integration with Steam. By default the Steam Deck uses KDE as the desktop mode. This inherently means it receives updates from Valve in terms of desktop related fixes, and they are actively working with KDE developers to bring updates to KDE (https://www.reddit.com/r/kde/comments/t2184m/is_there_any_cooperation_with_valve_and_kde/)
```

https://nobaraproject.org/category/changelog/

tl;dr? Plasma provides:

- Variable Refresh Rate (VRR) support.
- Virtual Reality (VR) support.
- Fractional scaling for more fine-tuned scaling.
- Drag and drop with the file manager.
- Feature development from Valve.
